Files/Filesystem/ListedItem.cs

```diff
@@ -248,7 +248,7 @@ public static string GetFriendlyDateFromFormat(DateTimeOffset d, string returnFo
             }
         }
 
-        private ObservableCollection<FileProperty> fileDetails = new ObservableCollection<FileProperty>();
+        private ObservableCollection<FileProperty> fileDetails;
```

Comment:
Looks like this change causes 
![image](https://user-images.githubusercontent.com/20501502/106396658-0d48dd00-6412-11eb-8a3a-be1e41687c38.png)

Reply:
Thanks, fixed.